open-banking-accelerator/internal-apis/internal-webapps/com.wso2.openbanking.authentication.webapp/src/main/webapp/js/respond.min.js

```diff
@@ -0,0 +1,5 @@
+/*! Respond.js v1.4.2: min/max-width media query polyfill * Copyright 2013 Scott Jehl
+ * Licensed under https://github.com/scottjehl/Respond/blob/master/LICENSE-MIT
```

Comment:
do we need to check the license here

Reply:
IMO We can skip these files as these are third-party JS libraries that we have embedded to source just to reduce load time.